usaspending_api/api_contracts/contracts/v2/disaster/agency/count.md

```diff
@@ -41,6 +41,8 @@ This endpoint returns a count of Agencies
 
 ## Filter (object)
 + `def_codes` (required, array[DEFC], fixed-type)
++ `award_type_codes` (optional, array[AwardTypeCodes], fixed-type)
+    If ANY award type codes are provided, count will be calculated from file C data, while if this parameter is not provided count will be calculated from file B. 
```

Comment:
I'm don't think if "File C" and "File B" by themselves are in the vocabulary of the USAspending API. Hopefully we have not used those excessively for public API docs. It should refer to the domain that USAspending uses. 

Additionally, the count doesn't exactly come from File C, it comes from the `awards` records linked in File C. 

Maybe:
> ... count will be calculated form the distinct set of toptier agencies, whose subtier agencies funded awards linked to `FinancialAccountsByAwards` records (which are derived from DABS File C). While if this parameter is not provided count will be calculated from the distinct set of toptier agencies "owning" appropriations accounts used in funding this disaster (from `FinancialAccountsByProgramActivityObjectClass` records, which are derived from DABS File B).

Reply:
Also, minor type `form` -> `from`

Reply:
Also, need to include:

```
## AwardTypeCodes (enum[string])
List of procurement and assistance award type codes supported by USAspending.gov

### Members
+ `02`
+ `03`
+ `04`
+ `05`
+ `06`
+ `07`
+ `08`
+ `09`
+ `10`
+ `11`
+ `A`
+ `B`
+ `C`
+ `D`
+ `IDV_A`
+ `IDV_B_A`
+ `IDV_B_B`
+ `IDV_B_C`
+ `IDV_B`
+ `IDV_C`
+ `IDV_D`
+ `IDV_E`
```

in the docs now